src/layout/Summary2/CommonSummaryComponents/SingleValueSummary.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import { ErrorMessage, Label, Paragraph } from '@digdir/designsystemet-react';
+import cn from 'classnames';
+
+import { Lang } from 'src/features/language/Lang';
+import { EditButton } from 'src/layout/Summary2/CommonSummaryComponents/EditButton';
+import classes from 'src/layout/Summary2/CommonSummaryComponents/SingleValueSummary.module.css';
+import type { BaseValidation } from 'src/features/validation';
+import type { LayoutNode } from 'src/utils/layout/LayoutNode';
+
+type SingleValueSummaryProps = {
+  title: React.ReactNode;
+  errors: BaseValidation[];
+  componentNode: LayoutNode;
+  displayData?: string;
+};
+
+export const SingleValueSummary = ({ title, errors, componentNode, displayData }: SingleValueSummaryProps) => (
+  <div className={classes.inputSummaryItem}>
+    <div className={classes.labelValueWrapper}>
+      <Label weight={'regular'}>{title}</Label>
+      <Paragraph
+        asChild
+        className={cn({
+          [classes.error]: errors.length > 0,
+          [classes.emptyValue]: !displayData,
+          [classes.formValue]: displayData,
+        })}
+      >
+        <span>
+          {!displayData && <Lang id={'general.empty_summary'}></Lang>}
```

Comment:
```suggestion
          {!displayData && <Lang id={'general.empty_summary'} />}
```

I'm sure prettier or eslint does this for us automatically? 🤔 I saw this at least one other place as well.